nilearn/_utils/estimator_checks.py

```diff
@@ -926,8 +995,6 @@ def check_surface_masker_fit_with_mask(estimator):
     estimator.mask_img = mask_img
     estimator.fit(imgs)
 
-    assert_polydata_equal(mask_img.data, estimator.mask_img_.data)
```

Comment:
removing as mask_img_ may now be different from mask_img (it may be a binarized version of it)

Reply:
Is it documented somewhere?

Reply:
Yup. I need to update the doc string for the attributes.
Some were wrong to start with. Now they would be wrong er.

Reply:
updated doc strings for the mask_img_ attributes